dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
These [other variables]( https://github.com/dask/dask/blob/175ae5a91ac09940c1065ea007a6a93a95d39207/dask/array/tests/test_fft.py#L20-L22 ) were already defined and use in the other tests. I used them to be consistent and to ensure the wrappers were doing the correct thing. I would like to keep this change focused on the task at hand.